src/Components/FormsCard/FormsCard.tsx

```diff
@@ -1,10 +1,58 @@
-import React from 'react';
-import { View, Text } from 'react-native';
+import { router } from 'expo-router';
+import React, { useEffect, useState } from 'react';
+import { View, Text, TouchableOpacity } from 'react-native';
+
+import styles from './styles';
+import RightCaret from '../../../assets/right-caret.svg';
+import { getFeaturedForm } from '../../app/(BottomTabNavigation)/AllCases/Forms/utils';
+import { Case, Form } from '../../types/types';
+import FormListItem from '../FormListItem/FormListItem';
+
+export default function FormsCard(caseData: Case) {
+  const [featuredForm, setFeaturedForm] = useState<Form>();
+
+  const getForm = async () => {
+    const formData = await getFeaturedForm(
+      caseData.id,
+      caseData.featuredFormName,
+    );
+    if (formData) {
+      setFeaturedForm(formData);
+    }
+  };
+
+  useEffect(() => {
+    if (caseData.formCount > 0) {
+      getForm();
+    }
+  });
```

Comment:
Let's add a dependency array here so that the query does not constantly run

Reply:
thanks, this resolved it!